opm/simulators/wells/BlackoilWellModelConstraints.cpp

```diff
@@ -593,7 +495,7 @@ updateGroupIndividualControl(const Group& group,
                              std::map<std::string, std::pair<std::string, std::string>>& closed_offending_wells,
                              GroupState<Scalar>& group_state,
                              WellState<Scalar, IndexTraits>& well_state,
-                             DeferredLogger& deferred_logger) const
+                             DeferredLogger& deferred_logger)
 {
```

Comment:
Why did you need to make this non const

Reply:
Good catch! I do not remember. It was in the first commit I think this became non-const.